odoo/addons/base/base_data.xml

```diff
@@ -1,6 +1,20 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
     <data noupdate="1">
+      <!-- khmer language has the wrong code in res.lang.csv
+      We put the good one here to have the noupdate tag and avoid
+      the error (fix only for v11.0) -->
```

Comment:
instead of writing in which version it applies, specify in which version it should be removed (saas-11.2)